server/routerlicious/packages/test-utils/src/messageFactory.ts

```diff
@@ -84,7 +84,8 @@ export class MessageFactory {
             referenceSequenceNumber,
             traces: [],
             type,
-        };
+            compression: undefined,
```

Comment:
This should not be needed, right?
It probably does not matter that much, but I observed similar change to be costly in tight loop:

https://github.com/microsoft/FluidFramework/pull/12058/files#diff-4955d937755632e07ceb6cff6c215904ae4e7cc8ceacdbf801868e3188760594R372

Reply:
We get two failing tests if I don't include this change where the tests are checking that the socket receives the same message that the document factory generates. I could change the tests instead (there are only two) or I can keep this change and the tests will pass. Which do we prefer?

Reply:
Or, that's test code - I guess it does not matter :) Putting undefined is fine here